dropwizard/service/src/main/java/org/apache/polaris/service/dropwizard/PolarisApplication.java

```diff
@@ -301,7 +303,12 @@ protected void configure() {
                 // manager
                 bindFactory(PolarisMetaStoreManagerFactory.class)
                     .in(RealmScoped.class)
+                    .named("delegateGrantManager")
```

Comment:
The name looks too generic. It does not convey the purpose of the beam. Also, the system could potentially have multiple delegates. How about `primaryGrantManager`, `persistingGrantManager`?